api4/system.go

```diff
@@ -71,6 +72,13 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.System.Handle("/onboarding/complete", api.APISessionRequired(getOnboarding)).Methods("GET")
 	api.BaseRoutes.System.Handle("/onboarding/complete", api.APISessionRequired(completeOnboarding)).Methods("POST")
 	api.BaseRoutes.System.Handle("/schema/version", api.APISessionRequired(getAppliedSchemaMigrations)).Methods("GET")
+
+	// GET /api/v4/events/topics
```

Comment:
```suggestion
	// GET /api/v4/system/events/topics
```

This applies to similar occasions as well.